account_fiscal_position_rule/README.rst

```diff
@@ -0,0 +1,44 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
New Readme template in OCA:
* [https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)
* [https://github.com/OCA/maintainer-tools/tree/master/template/module/readme](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)